--- amrwb-6.0.0/configure.ac.orig	1970-01-01 01:00:00.000000000 +0100
+++ amrwb-6.0.0/configure.ac	2006-07-02 00:04:57.811644000 +0200
@@ -0,0 +1,8 @@
+AC_INIT([amrwb], [6.0.0])
+AC_CONFIG_SRCDIR([typedef.h])
+AM_INIT_AUTOMAKE
+
+AC_PROG_CC
+AC_PROG_LIBTOOL
+
+AC_OUTPUT([Makefile])
--- amrwb-6.0.0/Makefile.am.orig	1970-01-01 01:00:00.000000000 +0100
+++ amrwb-6.0.0/Makefile.am	2006-07-02 00:07:12.108037000 +0200
@@ -0,0 +1,22 @@
+lib_LTLIBRARIES = libamrwb.la
+
+libamrwb_la_SOURCES = \
+	enc_acelp.c enc_dtx.c enc_gain.c enc_if.c enc_lpc.c enc_main.c enc_rom.c enc_util.c \
+	dec_acelp.c dec_dtx.c dec_gain.c dec_if.c dec_lpc.c dec_main.c dec_rom.c dec_util.c \
+	if_rom.c
+
+libamrwb_la_LIBADD = -lm
+
+amrwbincludedir = $(includedir)/amrwb
+amrwbinclude_HEADERS = \
+	dec.h dec_acelp.h dec_dtx.h dec_gain.h dec_if.h dec_lpc.h dec_main.h dec_util.h \
+	enc.h enc_acelp.h enc_dtx.h enc_gain.h enc_if.h enc_lpc.h enc_main.h enc_util.h \
+	if_rom.h typedef.h
+
+noinst_PROGRAMS = amrwb-decoder amrwb-encoder
+
+amrwb_decoder_SOURCES = decoder.c
+amrwb_decoder_LDADD = libamrwb.la
+
+amrwb_encoder_SOURCES = encoder.c
+amrwb_encoder_LDADD = libamrwb.la
