--- amrwb-6.0.0/typedef.h.orig	2003-08-27 00:00:02.000000000 +0200
+++ amrwb-6.0.0/typedef.h	2006-07-01 23:55:02.246423500 +0200
@@ -6,13 +6,15 @@
 #ifndef typedef_h
 #define typedef_h
 
+#include <inttypes.h>
+
 /* change these typedef declarations to correspond with your platform */
-typedef char Word8;
-typedef unsigned char UWord8;
-typedef short Word16;
-typedef unsigned short UWord16;
-typedef long Word32;
+typedef int8_t Word8;
+typedef uint8_t UWord8;
+typedef int16_t Word16;
+typedef uint16_t UWord16;
+typedef int32_t Word32;
 typedef double Float64;
 typedef float Float32;
 
-#endif
\ Brak znaku nowej linii na koñcu pliku
+#endif
